packages/manager-react-components/package.json

```diff
@@ -36,8 +36,8 @@
   },
   "dependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@ovhcloud/ods-components": "18.0.0",
-    "@ovhcloud/ods-themes": "18.0.0",
+    "@ovhcloud/ods-components": "18.3.0",
+    "@ovhcloud/ods-themes": "18.3.0",
```

Comment:
To avoid bumping the library every time ODS release a new version, I think we can move ODS to peerdep and use '^'

Reply:
👍  Okay, but after make some test, we had to let ods in devDependencies and peerDependencies. To run vitest test in MRC